src/base.js

```diff
@@ -225,14 +225,14 @@ VRDisplay.prototype.addFullscreenListeners_ = function(element, changeHandler, e
   if (changeHandler) {
     element.addEventListener('fullscreenchange', changeHandler, false);
     element.addEventListener('webkitfullscreenchange', changeHandler, false);
-    element.addEventListener('mozfullscreenchange', changeHandler, false);
+    document.addEventListener('mozfullscreenchange', changeHandler, false);
```

Comment:
be sure to do https://github.com/borismus/webvr-polyfill/pull/39/files#diff-d7e697eff3913f1acaacac8002c3b05eR250